### ISSUE TITLE
Update code base to QISKit 0.7

### ISSUE THREAD

**Issue by kevinawilson** [Apress/practical-quantum-computing#1]
The code in the book seems to use QISKit 0.5 or earlier. Starting in 0.6, the class QuantumProgram was deprecated. Because programs in the book and in this repository use QuantumProgram, the code throws errors.

It would be nice if the code could be updated to work with the most recent version of QISKit, especially since this is such a new book.

**Comment by richardawe** [comment]
I totally agree with @kevinawilson . I have tried replicating most of the examples and it is not working. There is a need to explain how QuantumProgram can be adapted in the codes to yield the same results or better.

**Comment by barryburd** [comment]
I'd like to know how to login to the Qiskit REST API since the most recent changes were made.

**Comment by hack-r** [comment]
I agree. Most code and URIs are out of date. My understanding is that a new edition is in the works.
